Module4_Labs/Lab2_Doubly_Linked_List/Cards/211.md

```diff
@@ -4,7 +4,7 @@
 
 <!--concepts={The Linked List}-->
 
-The function `printList(node)` will help you to visualize the list. It does this by traversing through the list and printing each node's data value. It will halt when `node` equals `None`.
+The function `printList(self, node)` will help you to visualize the list. It does this by traversing through the list and printing each node's data value. It will halt when `node` equals `None`.
```

Comment:
remind them that when node = "None" it is when they have reached the end of the list.